scripts/is/intg/helper-scripts/consolidate_db_is.sh

```
@@ -0,0 +1,247 @@
+  done
+
+  # Append consent schema into the identity SQL file
+  cat "$SCRIPT_LOCATION/consent/oracle.sql" >> "$WSO2_PRODUCT_VERSION_SHORT/is_oracle_identity.sql"
+  echo "" >> "$WSO2_PRODUCT_VERSION_SHORT/is_oracle_identity.sql"
+
+elif [ $DB_ENGINE = "db2-se" ]; then
+  # Create database creation script
+  db_create_file="$WSO2_PRODUCT_VERSION_SHORT/is_db2_db_create.sql"
+  touch "$db_create_file"
+  echo "-- DB2 Database Creation Script" > "$db_create_file"
+  echo "BEGIN" >> "$db_create_file"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)